integration_tests/suite/test_call_log_generation.py

```diff
@@ -1274,3 +1282,85 @@ def test_conference_has_a_destination_name(self):
+ CHAN_END         | 2021-10-06 13:55:58.692876-04 | Alice pcm | 1130    | 1130    | adhoc_conference | convert_to_stasis | PJSIP/Y4sSJpnV-00000011 | AppDial    | (Outgoing Line)                                                                                                       | 1633542835.29 | 1633542834.28 |                                                 |
+ BRIDGE_EXIT      | 2021-10-06 13:55:58.718222-04 | fb user2  | 1802    | 1802    | adhoc_conference | convert_to_stasis | PJSIP/auc6927d-00000013 | Stasis     | adhoc_conference,abc9c96e-2a96-43b6-9784-bb3e70c587e5                                                                 | 1633542906.31 | 1633542834.28 |                                                 | {"bridge_id":"abc9c96e-2a96-43b6-9784-bb3e70c587e5","bridge_technology":"simple_bridge"}
+ HANGUP           | 2021-10-06 13:55:58.718839-04 | fb user2  | 1802    | 1802    | adhoc_conference | convert_to_stasis | PJSIP/auc6927d-00000013 | AppDial    | (Outgoing Line)                                                                                                       | 1633542906.31 | 1633542834.28 |                                                 | {"hangupcause":16,"hangupsource":"","dialstatus":""}
+ CHAN_END         | 2021-10-06 13:55:58.718839-04 | fb user2  | 1802    | 1802    | adhoc_conference | convert_to_stasis | PJSIP/auc6927d-00000013 | AppDial    | (Outgoing Line)                                                                                                       | 1633542906.31 | 1633542834.28 |                                                 |
+ LINKEDID_END     | 2021-10-06 13:55:58.718839-04 | fb user2  | 1802    | 1802    | adhoc_conference | convert_to_stasis | PJSIP/auc6927d-00000013 | AppDial    | (Outgoing Line)                                                                                                       | 1633542906.31 | 1633542834.28 |                                                 |
+'''
+    )
+    def test_adhoc_conference(self):
+        self.confd.set_users(
+            MockUser(USER_1_UUID, USERS_TENANT, line_ids=[1]),
+            MockUser(USER_2_UUID, USERS_TENANT, line_ids=[2]),
+            MockUser(USER_3_UUID, USERS_TENANT, line_ids=[3]),
+        )
+        self.confd.set_lines(
+            MockLine(
+                id=1,
+                name='pa9pkxh5',
+                users=[{'uuid': USER_1_UUID}],
+                tenant_uuid=USERS_TENANT,
+                extensions=[{'exten': '1801', 'context': 'internal'}],
+            ),
+            MockLine(
+                id=2,
+                name='auc6927d',
+                users=[{'uuid': USER_2_UUID}],
+                tenant_uuid=USERS_TENANT,
+                extensions=[{'exten': '1802', 'context': 'internal'}],
+            ),
+            MockLine(
+                id=2,
```

Comment:
same id as  previous line, could this be problematic?

Reply:
fixed thanks